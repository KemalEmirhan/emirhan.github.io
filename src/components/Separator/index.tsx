import React from 'react';

const Separator = () => {
  return <hr className='separator bg-black hidden sm:block w-full h-1' />;
};

export default Separator;
