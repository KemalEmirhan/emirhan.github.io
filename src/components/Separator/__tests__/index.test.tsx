import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import Separator from '../';

describe('Separator', () => {
  it('should render', () => {
    const { container } = render(<Separator />);
    expect(container).toMatchSnapshot();
  });
});
