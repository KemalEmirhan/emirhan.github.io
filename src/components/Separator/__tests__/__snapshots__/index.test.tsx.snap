// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Separator > should render 1`] = `
<div>
  <hr
    class="separator max-w-6xl mx-auto bg-black w-full h-1 undefined"
  />
</div>
`;
